pages/plans/[id].js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Is this import required? It wasn't required on previous pages

Reply:
`React` is technically required on any page that uses JSX since after the JSX gets transpiled it ends up as `React.createElement('div', ...` calls. Next does some magic to "globally" import React though so shouldn't be required explicitly and so we can likely remove this.